goss-testing/tests/ncn/goss-k8s-lb-status.yaml

```diff
@@ -0,0 +1,38 @@
+#
+# MIT License
+#
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+{{ $scripts := .Env.GOSS_BASE | printf "%s/scripts" }}
+{{ $logrun := $scripts | printf "%s/log_run.sh" }}
+command:
+    {{ $testlabel := "check_k8s-lb-status" }}
+    {{$testlabel}}:
+        title: Verify that Load Balancer for Kubernetes is running
+        meta:
+            desc: If this test fails, verify that the load balancer is properly configured. See troubleshooting/known_issues/check_system_management_monitoring_tools.md in the CSM documentation for additional remediation steps.
+        exec: |-
+            "{{$logrun}}" -l "{{$testlabel}}" \
+                kubectl get service -A | grep Load|grep -i pending
```

Comment:
I have run this check on a bunch of systems and never seen it pass. There are always a couple of pendings. Specifically, I usually see these two:

```text
istio-system        istio-ingressgateway-chn                                LoadBalancer   10.17.172.129   <pending>       80:32169/TCP,443:31786/TCP                                 7d12h
services            cray-oauth2-proxies-customer-high-speed-ingress         LoadBalancer   10.25.120.69    <pending>       443:31655/TCP
```

Where have you run this test and seen it pass?